packages/presentation/src/PostMessageReleases.tsx

```diff
@@ -0,0 +1,40 @@
+import type {ClientPerspective} from '@sanity/client'
+import {memo, useEffect, type FC} from 'react'
+import {getVersionFromId, useClient} from 'sanity'
+import type {VisualEditingConnection} from './types'
+
+export interface PostMessageReleasesProps {
+  comlink: VisualEditingConnection
+  perspective: ClientPerspective | `bundle.${string}`
+  bundlesPerspective: string[]
+}
+
+const PostMessageReleases: FC<PostMessageReleasesProps> = (props) => {
```

Comment:
This component will be used only inside the studio right? In `PresentationTool` 

Maybe we could reuse here the releases from `import {useReleases} from 'sanity'` , what do you think?

Reply:
in that case, the studio will be responsible for providing the list of releases, which will change a bit in the near future. And you won't need to do this here. 

If it's inside the context of the studio the hook should work

Reply:
Ah awesome — this works! I initially used a simple query as eventually we would like to try this out in custom overlay components and it was good research, but we don't need it here. Have updated :)